dotcom-rendering/src/layouts/StandardLayout.tsx

```diff
@@ -111,6 +113,17 @@ const StandardGrid = ({
 									'meta   border  body         right-column'
 									'.      border  .            right-column';
 						  `
+						: isMedia
+						? css`
+								grid-template-areas:
+									'title  border  headline     .'
+									'.      border  disclaimer   right-column'
```

Comment:
As we have no horizontal gap between rows, do we need a whole specific set of template areas or can we rely on the fact that this row will collapse when there isn’t anything in it?

Reply:
Not sure I follow, the main reason is to move the standfirst below the media